src/utils/wrappers.ts

```diff
@@ -427,16 +469,19 @@ export function handleWrappers(ast: Node) {
         }
       }
       for (const [, m] of modules) {
-        if (m.params.length === 3 && m.params[2].type === 'Identifier') {
+        if ('params' in m && m.params.length === 3 && m.params[2].type === 'Identifier') {
           const assignedVars = new Map();
-          walk(m.body, {
-            enter (node, maybeParent) {
+          walk((m.body as unknown) as ESNode, {
+            enter (_node, _maybeParent) {
+              const node = _node as Node;
+              const maybeParent = _maybeParent as Node | undefined;
```

Comment:
@guybedford What are your thoughts here? I'm pretty sure this is supposed to be a `Node` not `ESNode` and its surfacing a couple bugs below.

Reply:
The `walk` function here uses a different `Node` base type that has the `start` and `end` requirements as well. Typing this function properly is thus difficult due to the compat differences - can't use the base typing library you need to use another one.

Reply:
I’m not so much interested in the walk function but the enter handler. Can you talk a look at my most recent commit here to see how to resolve the error below?

Reply:
Sure, I've pushed up a fix.